style-transfer/README.md

```diff
@@ -0,0 +1,18 @@
+## WebNN Fast Style Transfer Example
+Fast neural style transfer is the process of taking the style of one image then applying it to the content of another image.
+This example shows how the ['Fast Style Transfer' models](https://github.com/acerwebai/VangoghCrazyWorld)
+from Acer in ONNX format may be implememented by using the WebNN API.
```

Comment:
It would be better to say "from Vangogh Crazy World project"

Reply:
Please also mention where the picture assets come from? What's the license of them?